Vagrantfile

```diff
@@ -10,7 +12,7 @@ Vagrant.configure("2") do |config|
     end
 
     # If you change this IP, also change the DNS server for the "web" VM.
-    adfs2016.vm.network "private_network", ip: "10.0.0.2"
+    adfs2016.vm.network "private_network", ip: "11.0.0.2"
```

Comment:
Note that this IP is not private (unlike the 10.0.0.0/8 block). Any reason why we can't use an adress in a private range?

Reply:
No, perhaps 10.1.* then. I was avoiding what was already being used in my scenario, and can be a wifi router range